main.go

```diff
@@ -141,14 +141,12 @@ func handleConflicts(conflicts map[int]lr1Items.RowConflicts, numSets int, cfg c
 	if len(conflicts) <= 0 {
 		return
 	}
-	switch {
-	case !cfg.AutoResolveLRConf():
-		fmt.Printf("Error: %d LR-1 conflicts\n", len(conflicts))
+	fmt.Printf("Error: %d LR-1 conflicts \n", len(conflicts))
```

Comment:
I guess its not really an `Error`, when cfg.AutoResolveLRConf() is true though.
You might have to do two different prints of the LR-1 conficts :(

Reply:
Done!